### PR TITLE
Add param to get

### PR DESCRIPTION
**Pull request by cbonaudo**

### CODE REVIEW

File: backend/src/genre/model.rs [hunk 1]
```diff
@@ -2,13 +2,13 @@ use serde::{Deserialize, Serialize};
 
 #[derive(Serialize, Deserialize)]
 pub struct Genre {
-    id: u8,
+    id: i32,
```
Comment (o0Ignition0o) [inline backend/src/genre/model.rs:5]: Would an id ever be negative ?

File: backend/src/routes.rs [hunk 2]
```diff
@@ -2,18 +2,25 @@ use crate::genre;
 use tide::Request;
 
 pub fn routes(app: &mut tide::server::Server<()>) {
-    app.at("/")
+    app.at("/genres")
```
Comment (o0Ignition0o) [inline backend/src/routes.rs:5]: I really like the way we separate routes here, nice! 👍

File: backend/src/routes.rs [hunk 3]
```diff
@@ -2,18 +2,25 @@ use crate::genre;
...
-        .delete(delete_genre);
+        .post(post_genre);
+    app.at("/genres/:id")
+        .get(get_genre_by_id)
+        .delete(delete_genre_by_id);
 }
 
-async fn get_genres(_cx: Request<()>) -> String {
-    genre::controller::get()
+async fn get_genres(_req: Request<()>) -> String {
```
Comment (o0Ignition0o) [inline backend/src/routes.rs:13]: would it make sense to create genres.rs and call genres::get genres::post etc ?

Comment (cbonaudo) [reply backend/src/routes.rs:13]: I had some trouble having genre.rs and genre/mod.rs cohabiting, so I went with all in genre/mod.rs, WYT

Comment (o0Ignition0o) [reply backend/src/routes.rs]: Looks perfect to me (splitting it further would make no sense for now)

FYI here's the documentation on how to deal with modules and files: https://doc.rust-lang.org/book/ch07-00-managing-growing-projects-with-packages-crates-and-modules.html

**Pull request merged**